sortinghat/cmd/unify.py

```diff
@@ -216,3 +243,66 @@ def __display_stats(self):
         self.display('unify.tmpl', processed=self.total,
                      matched=self.matched,
                      unified=self.total - self.matched)
+
+    def __matches_log(self):
```

Comment:
All these methods can be encapsulated inside an object. This object will save, remove or recover entries and it is independent of how the matches are stored under the hood.